pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -1699,7 +1699,15 @@
     private double managedLedgerCacheEvictionWatermark = 0.9;
     @FieldContext(category = CATEGORY_STORAGE_ML,
             doc = "Configure the cache eviction frequency for the managed ledger cache. Default is 100/s")
-    private double managedLedgerCacheEvictionFrequency = 100.0;
+    @Deprecated
+    private double managedLedgerCacheEvictionFrequency = 100;
+
+    @FieldContext(category = CATEGORY_STORAGE_ML,
+            doc = "Configure the cache eviction interval in milliseconds for the managed ledger cache."
+                    + " This configuration takes effect if bigger than 0,"
+                    + " otherwise, managedLedgerCacheEvictionFrequency does")
+    private long managedLedgerCacheEvictionIntervalMs = 0;
```

Comment:
Should this take effect by default? So that, we can safely remove `managedLedgerCacheEvictionFrequency` in future?

Reply:
Yeah, `managedLedgerCacheEvictionIntervalMs` should take effect.